### PR TITLE
Documenting OpenAI compatibility (and other docs tweaks)

### PR DESCRIPTION
**Pull request by jamesbraza**
Modernization of https://github.com/jmorganca/ollama/pull/661

- ~Closes https://github.com/jmorganca/ollama/issues/538~
- Upstreams more knowledge from https://github.com/jmorganca/ollama/issues/546
- Simplifies `brew install` to one line

### CODE REVIEW

**Comment by jamesbraza** [comment]
Cc @jmorganca

File: README.md [hunk 1]
```diff
@@ -203,9 +203,9 @@ Finally, in a separate shell, run a model:
 
 ## REST API
 
-See the [API documentation](docs/api.md) for all endpoints.
-
-Ollama has an API for running and managing models. For example to generate text from a model:
+Ollama has a REST API for running and managing models.
```
Comment (jmorganca) [inline README.md:206]: The other changes in this PR look great, however would it be possible to separate out the changes in the `REST API` section here? I'm spending some time today thinking about this section and the API docs and don't want to slow down merging this PR.

**Comment by jmorganca** [comment]
Thanks for the PR! Small comment on the API section otherwise this is looking good

File: README.md (hunk 1 above)
Comment (jamesbraza) [reply README.md:206]: Sure, done!  I felt the ordering can be improved a bit, so I left some parts of this change, but I removed the note about OpenAI compatibility.  Ready for your review again!

Comment (jmorganca) [reply README.md:206]: Looks great!

**Review by jmorganca — APPROVED** [review]
Thanks for the PR! 🚀

**Pull request merged**